File: event-archive/src/main/java/com/omnibus/archive/repository/RecommendationArchiveRepository.java
package com.omnibus.archive.repository;

import com.omnibus.archive.model.RecommendationByDay;
import com.omnibus.archive.model.RecommendationByDayKey;
import org.springframework.data.cassandra.repository.ReactiveCassandraRepository;

public interface RecommendationArchiveRepository
    extends ReactiveCassandraRepository<RecommendationByDay, RecommendationByDayKey> {
}

File: services/event-archive/src/main/kotlin/com/omnibus/archive/repository/RecommendationArchiveRepository.kt
package com.omnibus.archive.repository

import com.omnibus.archive.model.RecommendationByDay
import com.omnibus.archive.model.RecommendationByDayKey
import org.springframework.data.cassandra.repository.ReactiveCassandraRepository

interface RecommendationArchiveRepository : ReactiveCassandraRepository<RecommendationByDay, RecommendationByDayKey>

